runner/runner.go

```diff
@@ -122,11 +123,17 @@ func (r *Runner) Run() error {
 				cmds = append(cmds, aws.String(command))
 			}
 
+			env, err := awsKVPForEnv(os.LookupEnv, r.Environment)
```

Comment:
`awsKeyValuePairForEnv` I reckon, otherwise it's like being smashed in the face with acronyms.

Reply:
Updated